Jenkinsfile

```diff
@@ -81,7 +81,7 @@ pipeline {
 }
 
 void publishArtifact() {
-    def current_tag = sh(returnStdout: true, script: "git tag --sort version:refname | tail -1").trim()
+    def current_tag = sh(returnStdout: true, script: "git tag --sort version:refname | sed '/v/d' | tail -1").trim()
```

Comment:
Remind me why we're doing this? Are our tags just numbers in git, rather than vM.m.p ?

Reply:
There was one release where we prepended v to the tag number, which breaks the sorting.